frontends/web/src/routes/account/send/send.tsx

```diff
@@ -501,6 +501,10 @@ class Send extends Component<Props, State> {
       address = url.pathname;
       if (this.isBitcoinBased()) {
         amount = url.searchParams.get('amount') || '';
+        if (amount && this.state.btcUnit === 'sat') {
+          // convert expected amount in BTC to sat
+          amount = `${Number(amount) * 100_000_000}`;
```

Comment:
feels like a job for the backend :thinking: feeling  a bit uncomfortable having conversion code exist both in the backend and frontend. the backend already has all the functions etc, and will also not suffer from any sort of floating point problems that could come from `Number(...)`.

Reply:
👍  yep makes sense.